themes/bootstrap3/js/common.js

```diff
@@ -537,10 +537,23 @@ function resetCaptcha($form) {
 }
 
 function bulkFormHandler(event, data) {
-  if ($('.checkbox-select-item:checked,checkbox-select-all:checked').length === 0) {
+  let numberOfSelected = $('.checkbox-select-item:checked').length;
```

Comment:
Maybe worth changing to `document.querySelector` to reduce jQuery usage while we're changing the line anyway.

Reply:
done